addons/mail/static/src/components/message_action_list/message_action_list.xml

```diff
@@ -2,21 +2,23 @@
 <templates>
     <t t-name="mail.MessageActionList" owl="1">
         <div class="o_MessageActionList d-flex" t-on-click="actionList.onClick">
```

Comment:
You forgot to guard this call, I guess it is only an issue if there is a click during the bad frame, but better be safe.

Reply:
If I can guard the root element, I also can remove the `<t t-if>` below it.

Reply:
You can't guard the root element because there must be always one, but because of that you need to guard usage of the record in the dynamic attributes of the root element.